tools/mason/Makefile

```diff
@@ -32,39 +32,19 @@ linkFile=$(bdir)/mason
 
 MASON_SOURCES=*.chpl
 
-all: mason
+all: $(bdir)/mason
 
 
-mason: $(MASON_SOURCES)
+$(bdir)/mason: $(MASON_SOURCES)
```

Comment:
I think this will not work. We run `make mason` in the home directory, and this change is modifying the target name `mason`.

Reply:
`make mason` in the home directory actually cd's to the mason directory and then runs `make` and then `make install`:

https://github.com/chapel-lang/chapel/blob/5c352439c60445decc130f4200f588d6433c8f98/Makefile#L151-L152

The `mason` target is not used there. However I changed this PR to leave `mason` and `install` targets that just run this target.

Reply:
Oops! I got confused between the two Makefiles! Thanks for explaining.